parameters.json.example

```diff
@@ -0,0 +1,7 @@
+{
+  "url": "https://sgwdev.behoerden-serviceportal.de",
+  "user": "myUser",
+  "duplicateProcesskeyAction": "ERROR",
```

Comment:
Ich glaube ein besserer default wäre hier `UNDEPLOY`, oder? Der Befehl wird ja primär während der Entwicklung genutzt in der das Überschreiben des aktuelle deployten Prozesses häufiger passiert als ein Deployment auf das Live-System :)